File: Chitunion/Web(java)/src/main/java/com/xyauto/domain/IpTitleInfo.java
package com.xyauto.domain;

import java.util.Date;

import lombok.Data;

@Data
public class IpTitleInfo {
	private Integer recId;
	private Integer pIp;
	private Integer subIp;
	private Integer relation;
	private Integer titleId;
	private Integer status;
	private Date createTime;
	private Integer createUserId;
	private String thisTitleName;
	private String pTitleName;
	private String subTitleName;
	private String relationName;
	private String showStr;

	public String getShowStr() {
		StringBuilder sb = new StringBuilder();
		if (!"".equals(this.pTitleName) && null != this.pTitleName) {
			sb.append(this.pTitleName + "-");
		}
		if (!"".equals(this.relationName) && null != this.relationName) {
			sb.append(this.relationName + "-");
		}
		if (!"".equals(this.subTitleName) && null != this.subTitleName) {
			sb.append(this.subTitleName + "-");
		}
		if (!"".equals(this.thisTitleName) && null != this.thisTitleName) {
			sb.append(this.thisTitleName);
		}
		return sb.toString();
	}
}

File: Chitunion/Web(code refactoring)/XYAuto.ChiTu2018/XYAuto.ChiTu2018.Entities/Chitunion2017/IPTitleInfo.cs
namespace XYAuto.ChiTu2018.Entities.Chitunion2017
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("IPTitleInfo")]
    public partial class IPTitleInfo
    {
        [Key]
        public int RecID { get; set; }

        public int? PIP { get; set; }

        public int? SubIP { get; set; }

        public int? Relation { get; set; }

        public int? TitleID { get; set; }

        public int? Status { get; set; }

        public DateTime? CreateTime { get; set; }

        public int? CreateUserID { get; set; }
    }
}
